cron/manifest.yaml

```
@@ -0,0 +1,15 @@
+---
+applications:
+  - name: cron
+    no-route: true
+    health-check-type: process
+    memory: 64MB
+    instances: 1
+    buildpacks:
+      - https://github.com/cloudfoundry/apt-buildpack
+      - binary_buildpack
+    command: |
+      ./install.sh
+      ./supercronic crontab 2>&1
```

Log messages should be going to `stderr` (file descriptor 2), while program output should be sent to `stdout` (file descriptor 1). I would expect that what we'll see most of the time from these jobs is log messages, correct?
```suggestion
      ./supercronic crontab
```

---

not needed